OldSrc/_scpcb1.4.0/oldRoomsAndWorldgen/SCPCB_Map/EntranceZone/hll_plain_2_a.as

```diff
@@ -0,0 +1,9 @@
+class hll_plain_2_a : Room {
```

Comment:
Again, old code is already backed up by git, duplicating it here is unnecessary.

Reply:
I wanted to be careful about deleting anything you may have wanted to retain.
Doing this let me restructure the angelscript code without major losses